src/fides/api/api/v1/endpoints/health.py

```diff
@@ -169,19 +186,23 @@ async def workers_health() -> Dict:
         },
     },
 )
-async def health() -> Dict:
+async def health(db: Session = Depends(get_db)) -> Dict:
```

Comment:
this is a no-no! notice that we've explicitly split out the more _involved_ health checks into their own endpoints to keep the core health check as lightweight as possible. we shouldn't introduce a db dependency into this core health check - instead we should either tack this on to the database health check (which already has a db dependency) or make it its own health check endpoint (or some other endpoint).

we've had issues in the past where the core app health check was slow to respond, which can exacerbate deployment debugging and maybe even contribute to general performance issues in deployed environments. happy to talk thru that more if it's not clear (i'm sure @RobertKeyser has some battle wounds he can share!) but the important thing here is keeping the core health check endpoint as lightweight as possible 👍

Reply:
Moved the email check out of the health check